lib/elixir/lib/application.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Application do
   @moduledoc """
-  A module for working with applications and defining application callbacks.
+  Convenience functions for working with applications and defining
```

Comment:
Can you please send a PR for the long ones, those would be great. But let's not change the ones under the proper length please. :)